LEN/deck.json

```diff
@@ -859,6 +859,16 @@
                     "note_model_uuid": "0f3f52aa-931b-11eb-b01b-04d9f5f4bc98",
                     "tags": []
                 },
+                {
+                    "__type__": "Note",
+                    "fields": [
+                        "Wie berechnet man die Gesamtimpedanz zweier in Reihe geschalteten Impedanzen?",
+                        "\\(Z_{ges} = Z_1+Z_2\\)"
```

Comment:
Die karte war auch fehlerhaft, ich glaube die hast du aber schon gelöscht, ich habe sie jetzt mal korrigiert. gelöschte karten werden leider nicht synchronisiert, deshalb hatte ich noch die fehlerhafte karte

Reply:
Ja, du hast sie in commit e990099 gelöscht, das wird aber leider nicht synchronisiert